src/integrationTest/resources/application.yml

```diff
@@ -1,9 +1,18 @@
 spring:
+
+  jpa:
+    show-sql: true
+    properties:
+      hibernate:
+        enable_lazy_load_no_trans: true
```

Comment:
This is clearly discrimination. Why does everyone but me get to lazy load? :sob: